roles/etcd/templates/openssl.conf.j2

```diff
@@ -25,6 +25,9 @@ authorityKeyIdentifier=keyid:always,issuer
 [alt_names]
 DNS.1 = localhost
 {% for host in groups['etcd'] %}
+{# The address which etcd uses to access its members must be included in the SAN, otherwise etcd will fail with a TLS error upon startup. #}
+DNS.{{ counter["dns"] }} = {{ hostvars[host]['etcd_access_address'] }}{{ increment(counter, 'dns') }}
```

Comment:
This sounds a good idea but this var is an IP by default so this won't work in the default cSe so maybe there should be a check similar to this `hostvars[host]['etcd_access_address'] | ansible.utils.ipaddr }}`? WDYT?

Reply:
Yeah, I saw failing tests complaining about it not being defined as well. I've added a guard.